src/main/kotlin/no/nav/sbl/sosialhjelp_mock_alt/datastore/utbetaling/UtbetalDataService.kt

```diff
@@ -0,0 +1,29 @@
+package no.nav.sbl.sosialhjelp_mock_alt.datastore.utbetaling
+
+import com.fasterxml.jackson.databind.DeserializationFeature
+import com.fasterxml.jackson.datatype.jsr310.JavaTimeModule
+import com.fasterxml.jackson.module.kotlin.jacksonObjectMapper
+import com.fasterxml.jackson.module.kotlin.readValue
+import com.fasterxml.jackson.module.kotlin.registerKotlinModule
+import no.nav.sbl.sosialhjelp_mock_alt.datastore.utbetaling.model.UtbetalData.Utbetaling
+import org.apache.commons.io.IOUtils
+import org.springframework.stereotype.Service
+import java.nio.charset.StandardCharsets
+
+@Service
+class UtbetalDataService {
+
+    fun getUtbetalingerFraNav(ident: String): List<Utbetaling> {
+        val utbetaling = getUtbetalingFromJsonFile("inntekt/navutbetalinger/sokos-utbetaltdata-ekstern-response.json")
```

Comment:
Dette funker fint til å starte med 👍 
Tenker at etter hvert burde vi sikkert få på plass støtten for å legge til mock-utbetalinger slik som gjøres i UtbetalingService i dag - https://github.com/navikt/sosialhjelp-mock-alt-api/blob/master/src/main/kotlin/no/nav/sbl/sosialhjelp_mock_alt/datastore/utbetaling/UtbetalingService.kt#L21.

Vet ikke hvor ofte navutbetalinger legges til manuelt ved testing i mock 🤷 
Men sikkert fint å bruke denne json-fila som "default"-utbetaling eller noe slikt for default-testbrukerne i mock-alt

Reply:
Det er nok lurt ja! Bare jeg som glemte bort at mock-alt faktisk også brukes i et mock miljø og ikke bare til lokal test. Noterer det ned på blokka med "oppryddingsoppgaver".